### ISSUE TITLE
Добавить замедленное воспроизведение в визуализаторе

### ISSUE THREAD

**Issue by Oreshnik** [sat2707/aicups#73]
Даже скорость 1 довольно высока, приходится все время прокручивать на стрелке. Хотелось бы, чтобы можно было выбрать множитель хотя бы 0.5 (и 0.1, если это не будет супер медленно).

**Comment by ivan-kudziev** [comment]
можно еще поставить на паузу кликнуть мышкой на временной ползунок и управлять перемоткой стрелками

**Comment by Oreshnik** [comment]
Да, спасибо, я так и делаю. Но хотелось бы просто медленный просмотр без необходимости постоянно жать кнопку.
